src/components/ConfirmDialog.tsx

```
@@ -0,0 +1,92 @@
+  const handleBackdropClick = (e: React.MouseEvent) => {
+    if (e.target === e.currentTarget) {
+      onCancel();
+    }
+  };
+
+  return (
+    <div
+      className="confirm-dialog__backdrop"
+      data-testid="dialog-backdrop"
+      onClick={handleBackdropClick}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)